app/models/allocated_offender.rb

```diff
@@ -22,6 +22,10 @@ def latest_movement_date
     @offender.latest_movement&.create_date_time&.to_date
   end
 
+  def location
+    @offender.cell_location
+  end
```

Comment:
I'm not sure this is needed, since we already delegate `cell_location` to the offender object.

This just seems to duplicate that – so `AllocatedOffender` has two methods which return the same value:
- `allocated_offender.location`
- `allocated_offender.cell_location`

We should be able to remove the `location` method and just keep the delegated method `cell_location`.